src/Common/src/Interop/Windows/Kernel32/Interop.LoadLibraryEx_IntPtr.cs

```diff
@@ -12,7 +12,7 @@ internal static partial class Kernel32
         internal const int LOAD_LIBRARY_AS_DATAFILE = 0x00000002;
         internal const int LOAD_LIBRARY_SEARCH_SYSTEM32 = 0x00000800;
 
-        [DllImport(Libraries.Kernel32, CharSet = CharSet.Unicode, EntryPoint = "LoadLibraryExW", SetLastError = true)]
```

Comment:
The CppCodegen failure is because of this diff. I forgot to upstream this, sorry. Submitted a pull request in the runtime repo.